src/components/Scheduler/InnerSchedulers/WeeklyScheduler.tsx

```diff
@@ -0,0 +1,97 @@
+/* tslint:disable:no-bitwise */
+import * as React from 'react';
+import * as NumericInput from 'react-numeric-input';
+import { IInnerSchedulerProps } from './InnerScheduler.Props';
+import { TextField } from '../../TextField/TextField';
+import { CheckboxList } from '../../CheckboxList/CheckboxList';
+import { DaysOfWeekEnum } from '../../../index';
+
+export class WeeklyScheduler extends React.Component<IInnerSchedulerProps, any> {
```

Comment:
Can you please use a type or empty type ({}) for state instead of any. With any we use TypeScript's type definitions.